tests/test_download.py

```diff
@@ -104,6 +104,7 @@ def test_generate_bulk_download_filtered(
         workflow_type=WorkflowActivityTypeEnum.metagenome_annotation.value,
     )
     metag.outputs.append(metag_output)
+    op1.outputs.append(metag_output)
```

Comment:
This makes sure that the test objects (omics processing `op1` and data object `metag_output`) are related by a row in the association table `omics_processing_output_association`. This table _used to_ represent only direct outputs from data generations (i.e. raw data), but now represents all data objects downstream of a particular data generation. Hopefully that makes sense.